lib/routers/task/index.js

```diff
@@ -4,6 +4,15 @@ const isUUID = require('uuid-validate');
 const { NotFoundError } = require('../../errors');
 const router = Router({ mergeParams: true });
 
+router.get('/related', (req, res, next) => {
+  return req.workflow.related({ query: req.query })
+    .then(response => {
+      // don't fall through to the routes below as they will capture 'related' as ':taskId'
+      return res.json(response.json);
```

Comment:
```suggestion
      return next('router');
```

Reply:
This is the recommended way to exit from a router. There is a canonical function for sending responses that adds metadata and structures the response, and adding other places where responses are send make that potentially fragile.